test/parser/decimal.spec.ts

```diff
@@ -11,7 +11,11 @@ describe( 'decimal parsing', () => {
       ['+.1', '=+.1'],
       ['+.1', '=+.1+.2'],
       ['=SUM(A1:A4, 0.3, .3)', '=SUM(B1:B4)'],
-      ['.1.4', '=..1']
+      ['.1.4', '=..1'],
+      ['1.', '=1.'],
+      ['1e1', '=1e1'],
+      ['1e+1', '=1e+1'],
+      ['1e-1', '=1e-1'],
```

Comment:
Will it work with function as well? And as a criterion?

```suggestion
      ['1e-1', '=1e-1'],
      ['=SUM(A1:A4, 1e-1, .1e+1)', '=IF(A1=1e10, TRUE, FALSE)'],
```

Reply:
yes, and those tests do not belong here.

Reply:
I actually don't think this is an useful test. One valid thing to test is SUMIF functionality

Reply:
Ok, thanks